src/generated/endpoints.ts

```diff
@@ -223,8 +223,12 @@ const Endpoints: EndpointsDefaultsAndDecorations = {
       "PUT /user/installations/{installation_id}/repositories/{repository_id}",
     ],
     checkToken: ["POST /applications/{client_id}/token"],
-    createContentAttachment: [
-      "POST /content_references/{content_reference_id}/attachments",
+    createContentAttachmentForRepo: [
+      "POST /repos/{owner}/{repo}/content_references/{content_reference_id}/attachments",
+      { mediaType: { previews: ["corsair"] } },
+    ],
+    createContentAttachmentForRepo: [
+      "POST /repos/{owner}/{repo}/content_references/{content_reference_id}/attachments",
```

Comment:
`createContentAttachmentForRepo` should be listed twice, and `createContentAttachment` shouldn't be removed